slack_sdk/http_retry/builtin_interval_calculators.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+from .jitter import Jitter, RandomJitter
+from .interval_calculator import RetryIntervalCalculator
+
+
+class FixedValueRetryIntervalCalculator(RetryIntervalCalculator):
```

Comment:
Just as a reference. Unused with the default setting. I should add some test for this

Reply:
If it's unused, can probably not worry about the tests. Unless you want to keep the coverage scores high 😆

Reply:
Haha, yeah, I always like better coverage!